briskedu/brisk-learning
# Change heuristic for number of edits

Summary:
Revision count is too coarse of a metric and will roll up into one revision per day in most situations.  The history view has child revisions called "Detailed Revisions" but this isn't available in the API.

Instead we'll use the number of edits which luckily is captured in the id of each revision.  The number of edits is increasingly meaningless to a teacher since it's based on the number of auto-saves, not anything they can see in the UI.  So I think we should just remove it since we're only using it to auto-expand.

Test Plan:
1. Confirm number increases by 2 if I make 2 changes over 5 seconds
2. Confirm number and cutoff seem reasonable by looking at a few docs
